.buildkite/pipeline.yml

```diff
@@ -20,6 +20,7 @@ steps:
     branches: master
     agents:
       queue: native
```

Comment:
These are build node tags
```yaml
  - name: ":docker: Tag Staging"
    command: bin/tag staging
    branches: master
    agents:
      queue: native
      docker: builder
```

All the other steps have been switched to `queue: "*"`